core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -900,6 +900,11 @@
 
   @BaseMessage("Not a valid input for REGEXP_REPLACE: ''{0}''")
   ExInst<CalciteException> invalidInputForRegexpReplace(String value);
+
+  @BaseMessage("Illegal error behavior ''{0}'' EXTRACTVALUE: document: ''{1}'', xpath expression:"
+      + " ''{2}''")
+  ExInst<CalciteException> illegalErrorBehaviorInExtractValueFunc(String value, String xpath,
+      String errorBehavior);
 }
```

Comment:
Both `illegal` and `error` are negative, which seems redundant. Do you mean illegal xml path?

Reply:
Actually I referred IllegalErrorBehaviorInJsonQueryFunc, I have incorporated the changes :)